lib/components/ChecklistCard/ChecklistCard.tsx

```diff
@@ -19,14 +19,24 @@ export default class ChecklistCard extends Component<ChecklistCardProps> {
     const { children, ...restProps } = this.props;
     return (
       <Card {...restProps}>
-        {React.Children.map(children, (child, i) => (
-          <Fragment>
-            {i > 0 && <Divider />}
-            {React.cloneElement(child as ReactElement<CheckboxProps>, {
-              variant: 'inChecklistCard'
-            })}
-          </Fragment>
-        ))}
+        {React.Children.map(children, (child, i) => {
+          if (typeof child !== 'object') {
```

Comment:
What's this guarding against specifically?

Reply:
Children can be primitive types, like string or number, which don't have a `props` property.